build.gradle

```diff
@@ -139,6 +164,72 @@ allprojects {
     }
 }
 
+def log4jVersion = '2.22.1'
+
+configurations {
+    distOnly
+}
+
+dependencies {
+    distOnly "org.apache.logging.log4j:log4j-core:${log4jVersion}"
+    distOnly "org.apache.logging.log4j:log4j-slf4j2-impl:${log4jVersion}"
+}
+
+tasks.register('dist') {
```

Comment:
I think we should let `dist` task depend on `jar` task, otherwise we will get a `can't load the aclService plugin` error because `eventmesh-security-acl-1.10.0-release.jar` is missing in `plugin/security/acl`.

Reply:
Nice catch!

Actually `eventmesh-security-acl` is not copied by the `dist` task, but the `installPlugin` task. I fixed the dependencies of the latter in 8d5cab712b496b5166980d9f103075eb8f2c31ca.